store.go

```diff
@@ -51,6 +52,40 @@ func StoreFrom(to, from GenericStorer) {
 	to.SetGenericStore(from.GenericStore())
 }
 
+// LoggerFunc is a function that takes a log message with some arguments and
+// logs it.
+type LoggerFunc func(string, ...interface{})
+
+// debugProxy is a database proxy that logs all SQL statements executed.
+type debugProxy struct {
+	logger func(string, ...interface{})
+	proxy  squirrel.DBProxy
+}
+
+func defaultLogger(message string, args ...interface{}) {
+	log.Printf("%s, args: %v", message, args)
+}
+
+func (p *debugProxy) Exec(query string, args ...interface{}) (sql.Result, error) {
+	p.logger(fmt.Sprintf("kallax: Exec: %s", query), args...)
+	return p.proxy.Exec(query, args...)
+}
+
+func (p *debugProxy) Query(query string, args ...interface{}) (*sql.Rows, error) {
+	p.logger(fmt.Sprintf("kallax: Query: %s", query), args...)
+	return p.proxy.Query(query, args...)
+}
+
+func (p *debugProxy) QueryRow(query string, args ...interface{}) squirrel.RowScanner {
+	p.logger(fmt.Sprintf("kallax: QueryRow: %s", query), args...)
+	return p.proxy.QueryRow(query, args...)
+}
+
+func (p *debugProxy) Prepare(query string) (*sql.Stmt, error) {
```

Comment:
Can't test this one. There's no method that exposes a direct `Prepare` on Store. (It's done automatically underneath by the StmtCacher).